src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1230,19 +1252,19 @@
 helps['network nic show'] = """
     type: command
     short-summary: Get the details of a network interface.
+    long-summary: To learn more about network interfaces in Azure visit https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-network-interface
```

Comment:
Would you find it useful if we had a specific kwarg for article links vice putting it in the long-summary, or does the long-summary suffice?  If it would be useful for the doc generation, we could definitely do that.  For example, maybe we could have a CLI command that would open any linked article in a browser?